test/scylla-scripts-pytest/run

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+cd $(dirname $0)
+pytest -o junit_family=xunit2 --confcutdir . $*
```

Comment:
Is this "--confcutdir ." needed both here and in pytest.ini? Isn't at least one of them (I was hoping both...) unnecessary?